po/ja.po

```diff
@@ -2098,7 +2098,7 @@ msgstr ""
 msgid ""
 "Let us jump into the simplest possible Rust program, a classic Hello World\n"
 "program:"
-msgstr ""
+msgstr "さっそく伝統的なHello Worldプログラムを確認していきましょう："
```

Comment:
"伝統的な" sounds like "traditional". How about "定番の"?

Also, "確認していきましょう" feels a little different from "jump into" because it conveys the nuance that the reader already knows Rust.
I'd write "さっそく一番シンプルなプログラムである定番のHello Worldからみてみましょう (or はじめましょう)"